input/projects/data/csharp-standard.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: C# Standard
+Contributor: Ecma International
+Web: https://www.ecma-international.org/
+---
+# C# language standard
+
+[TC49-TG2](https://www.ecma-international.org/task-groups/tc49-tg2/) writes the official C# standard. The scope is to standardize the syntax and semantics of a modern, component-based, general purpose, object oriented, and type-safe programming language called C# (pronounced C sharp).
+
+The committee Programme of work:
```

Comment:
Does P need to be capitalized here? I'd potentially expect "s" of "standard" to be capitalized if we're doing caps for things, but I'm not sure that "Programme" needs it...